java/arcs/core/storage/api/Entity.kt

```diff
@@ -40,8 +39,8 @@ interface EntitySpec<T : Entity> {
      */
     fun deserialize(data: RawEntity): T
 
-    /** Returns the corresponding [Schema] for the specified [Entity]. */
-    fun schema(): Schema
+    /** The corresponding [Schema] for the specified [Entity]. */
+    val SCHEMA: Schema
```

Comment:
Why changing this to all caps?

Reply:
This was already being generated, and is being used from other places. We were generating some strange code:

```
companion object {
  val SCHEMA = Schema(...)
}

override fun schema() = SCHEMA
```

This just cuts out the middle man

Reply:
What's the Kotlin Readability spec for singleton constants? Is it all caps like Java? I ask because Kotlin enums use uppercase first letter whereas in Java it's all caps.

Reply:
All caps: https://kotlinlang.org/docs/reference/coding-conventions.html#property-names